### ISSUE TITLE
Class inheritance fails on prefixed class

### ISSUE THREAD

**Issue by astatide** [chapel-lang/chapel#13086]
### Summary of Problem

When creating a subclass in a separate chapel file, class inheritance fails if the class type has a prefix.  If there's no conflict in the current namespace, `use` and then just naming the class is a suitable workaround.

### Steps to Reproduce

**Source Code:**

FILE a.chpl
```chapel
class A {proc blah {return 0; }}
```
FILE b.chpl
```chapel
use a;
class B: A {proc iAmAFailure {return 1; }}
```

**Compile command:**

**Execution command:**

**Associated Future Test(s):**

### Configuration Information

- Output of `chpl --version`:
```
chpl version 1.19.0.2c1fa771fb
Copyright (c) 2004-2019, Cray Inc.  (See LICENSE file for more details)
```
- Output of `$CHPL_HOME/util/printchplenv --anonymize`:
```
CHPL_TARGET_PLATFORM: darwin
CHPL_TARGET_COMPILER: clang
CHPL_TARGET_ARCH: x86_64
CHPL_TARGET_CPU: native
CHPL_LOCALE_MODEL: flat
CHPL_COMM: none
CHPL_TASKS: qthreads
CHPL_LAUNCHER: none
CHPL_TIMERS: generic
CHPL_UNWIND: none
CHPL_MEM: jemalloc
CHPL_ATOMICS: cstdlib
CHPL_GMP: gmp
CHPL_HWLOC: hwloc
CHPL_REGEXP: re2
CHPL_AUX_FILESYS: none
```
- Back-end compiler and version, e.g. `gcc --version` or `clang --version`:
```
Apple LLVM version 10.0.0 (clang-1000.11.45.5)
Target: x86_64-apple-darwin18.2.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
```
- (For Cray systems only) Output of `module list`:

**Comment by mppf** [comment]
Here is a reproducer:

``` chapel
module ModuleA {
  class A { }
}

module ModuleB {
  class B: ModuleA.A { }
}
```

results in compilation error `error: Illegal super class`

**Comment by ajoshpratt** [comment]
oooof, thanks.  Accidentally posted the version that works, not the one that fails.  More coffee is needed.

**Comment by lydia-duncan** [comment]
@astatide - looks like @DanilaFe resolved this with his #25491.  Thanks for reporting, and thanks Daniel for the fix!
